network-configuration/ssl/pom.xml

```diff
@@ -11,15 +11,40 @@
     </description>
 
     <parent>
-        <groupId>com.hazelcast.samples.network-configuration</groupId>
-        <artifactId>network-configuration</artifactId>
+        <groupId>com.hazelcast.samples</groupId>
```

Comment:
SSL and symmetric encryption code samples require EE so it sucks that the code samples were using OS and failing to start.

Reply:
Shouldn't we rather move it to the `enterprise` directory? Just thinking

Reply:
Yes, you're right, I'll move it.

Reply:
Moved :)